brokerapi/brokers/account_managers/sql_account_manager.go

```diff
@@ -165,11 +165,29 @@ func (sam *SqlAccountManager) pollOperationUntilDone(op *googlecloudsql.Operatio
 	return nil
 }
 
-func (b *SqlAccountManager) BuildInstanceCredentials(bindDetails map[string]string, instanceDetails map[string]string) map[string]string {
+func (b *SqlAccountManager) BuildInstanceCredentials(bindRecord models.ServiceBindingCredentials, instanceRecord models.ServiceInstanceDetails) (map[string]string, error) {
+	instanceDetails := instanceRecord.GetOtherDetails()
+	bindDetails := bindRecord.GetOtherDetails()
+
+	sid := instanceRecord.ServiceId
+	service_to_name, err := utils.MapServiceIdToName()
+	if err != nil {
+		return map[string]string{}, err
+	}
+
 	combinedCreds := utils.MergeStringMaps(bindDetails, instanceDetails)
```

Comment:
nit: move up to 171, code introduced at 172/176 splits the logical flow

Reply:
ack